application/src/main/java/org/opentripplanner/street/model/vertex/StationEntranceVertex.java

```diff
@@ -0,0 +1,29 @@
+package org.opentripplanner.street.model.vertex;
+
+public class StationEntranceVertex extends OsmVertex {
+
+  private final String code;
+  private final boolean accessible;
+
+  public StationEntranceVertex(double x, double y, long nodeId, String code, boolean accessible) {
```

Comment:
There is already a class called `TransitEntranceVertex`. I think we could probably use that instead of this. Although, I don't know if we should reserve the use of it just for pathways? Thoughts @leonardehrenfried ?